package.json

```diff
@@ -33,11 +33,12 @@
   "author": "godu",
   "license": "MIT",
   "devDependencies": {
+    "ava": "^1.0.0-beta.7",
```

Comment:
This should be a fixed dependency. SemVer doesn't cover pre-releases so this would pull in all subsequent betas, and they may contain breaking changes.

I suppose as a consequence we should do beta releases of this package too, at least until AVA hits 1.0.